mbed_app.json

```diff
@@ -411,7 +394,10 @@
             "target.macros_remove"                      : ["MBEDTLS_CONFIG_HW_SUPPORT"]
         },
         "RZ_A1H": {
-            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS_SW_TRNG.h\"", "MBEDTLS_ENTROPY_NV_SEED", "PAL_USE_HW_TRNG=0"],
+            "events.shared-eventsize"                   : 2048,
```

Comment:
no need to repeat the events here. Its already on top-section.

Reply:
Done.